### PR TITLE
Note on full HD and scalable video

### PR DESCRIPTION
**Pull request by jeffswartz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -256,7 +256,10 @@ The `OTNetworkTest()` constructor includes the following parameters:
     (`true`) or not (`false`, the default). Disabling scalable video
     was added in OpenTok.js version 2.24.7.
 
-  * `fullHd` (Boolean) -- (Optional) Allows publishing with a resolution of 1920x1080. If the camera does not support 1920x1080 resolution, OTNetworkTest.testConnectivity() method is rejected with  `UNSUPPORTED_RESOLUTION_ERROR` error.
+  * `fullHd` (Boolean) -- (Optional) Allows publishing with a resolution of 1920x1080 (1080p).
+    If the camera does not support 1920x1080 resolution, the `OTNetworkTest.testConnectivity()` method
+    is rejected with an `UNSUPPORTED_RESOLUTION_ERROR` error. For best results, you should also
+    set the `scalableVideo` option to `true` when testing full HD resolution.
```
Comment (jeffswartz) [inline README.md:262]: Should we add another comment: "Without `scalableVideo`set to `true`, the client may not reach full HD resolution during the test period?"

File: README.md [hunk 2]
```diff
@@ -425,7 +428,7 @@ following properties:
       For example, `'No camera was found.'`
     
     * `qualityLimitationReason` (String) --  Indicates the reason behind
-       the highest resolution tested failing. It can have values: `'cpu'` for CPU overload, `'bandwidth'` for insufficient network bandwidth, or value is `'null'` if there is no limitation.
+       the highest resolution tested failing. It can have values: `'cpu'` for CPU overload, `'bandwidth'` for insufficient network bandwidth, or value is `null` if there is no limitation.
```
Comment (jeffswartz) [inline README.md:431]: This is set to `null` in Safari. But it is set to `'none'` in Chrome.

Comment (vona-ben) [reply README.md:430]: @jeffswartz Fixed

File: README.md (hunk 1 above)
Comment (vona-ben) [reply README.md]: Absolutely updated the README

**Review by vona-ben — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -256,7 +256,9 @@ The `OTNetworkTest()` constructor includes the following parameters:
     (`true`) or not (`false`, the default). Disabling scalable video
     was added in OpenTok.js version 2.24.7.
 
-  * `fullHd` (Boolean) -- (Optional) Allows publishing with a resolution of 1920x1080. If the camera does not support 1920x1080 resolution, OTNetworkTest.testConnectivity() method is rejected with  `UNSUPPORTED_RESOLUTION_ERROR` error.
+  * `fullHd` (Boolean) -- (Optional) Allows publishing with a resolution of 1920x1080 (1080p).
+    If the camera does not support 1920x1080 resolution, the `OTNetworkTest.testConnectivity()` method
+    is rejected with an `UNSUPPORTED_RESOLUTION_ERROR` error. We highly recommend set `scalableVideo` setting to `true` when testing full HD resolution. Without `scalableVideo`, the client may not reach full HD resolution during the test period..
```
Comment (jeffswartz) [inline README.md:261]:
```suggestion
    is rejected with an `UNSUPPORTED_RESOLUTION_ERROR` error. We highly recommend set `scalableVideo` setting to `true` when testing full HD resolution. Without `scalableVideo`, the client may not reach full HD resolution during the test period.
```

**Pull request merged**